src/components/AssetsTable/index.jsx

```diff
@@ -159,9 +164,11 @@ export class AssetsTable extends React.Component {
   }
 
   getLoadingLockButton(asset) {
-    const classes = [FontAwesomeStyles.fa, FontAwesomeStyles['fa-spinner'], FontAwesomeStyles['fa-spin']];
+    // spinner classes are applied to the span to keep the whole button from spinning
```

Comment:
Haha, that would be pretty fun though

Reply:
Ha! I did find it quite amusing in fact! 😸